<launch>
  <arg name="rs_serial_n1"    			default=""/> 			<!-- Note: Replace with actual serial number -->
  <arg name="rs_serial_n2"    			default=""/> 			<!-- Note: Replace with actual serial number -->
  <arg name="camera1"              			default="camera1"/>		<!-- Note: Replace with camera name -->
  <arg name="camera2"              			default="camera2"/>		<!-- Note: Replace with camera name -->
  <arg name="tf_prefix_camera1"         default="$(arg camera1)"/>
  <arg name="tf_prefix_camera2"         default="$(arg camera2)"/>
  <arg name="initial_reset"             default="false"/>
  <arg name="reconnect_timeout"         default="6.0"/>

  <group ns="$(arg camera1)">
    <include file="$(find mesapro)/launch/rs_nodelet.launch.xml">
      <arg name="serial_no"             value="$(arg rs_serial_n1)"/>
      <arg name="tf_prefix"         		value="$(arg tf_prefix_camera1)"/>
      <arg name="initial_reset"         value="$(arg initial_reset)"/>
      <arg name="reconnect_timeout"     value="$(arg reconnect_timeout)"/>
    </include>
  </group>

  <group ns="$(arg camera2)">
    <include file="$(find mesapro)/launch/rs_nodelet.launch.xml">
      <arg name="serial_no"             value="$(arg rs_serial_n2)"/>
      <arg name="tf_prefix"		          value="$(arg tf_prefix_camera2)"/>
      <arg name="initial_reset"         value="$(arg initial_reset)"/>
      <arg name="reconnect_timeout"     value="$(arg reconnect_timeout)"/>
    </include>
  </group>

</launch>
